CODEOWNERS

```diff
@@ -14,7 +14,7 @@
 /Hunting%20Queries/ @timbMSFT @aprakash13 @ashwin-patil @petebryan @shainw @thmcelro

```

Comment:
I am checking on why this is and will fix once I discover there reason.

Reply:
i think this needs to be in quotes per syntax "/Hunting Queries/"